test/testutil.c

```diff
@@ -161,3 +161,13 @@ int strings_equal(const char *desc, const char *s1, const char *s2)
     }
     return 1;
 }
+
+/* TODO : hex decoder should be implemented */
+void hexdump(const unsigned char *a, size_t len)
+{
+    size_t i;
+
+    for (i = 0; i < len; i++)
+        fprintf(stderr, "%02x", a[i]);
+}
+
```

Comment:
Cleaning up all "hexdump" to use that one common routine is a separate PR, but let's not make it the situation worse with this one.